src/containers/CharacterProfile/index.js

```diff
@@ -19,18 +19,10 @@ import {
 import LinearGradient from 'react-native-linear-gradient';
 
 // components
-import ProfileBackDrop from '../../components/CharacterProfile/ProfileBackDrop';
-import ProfilePicture from '../../components/CharacterProfile/ProfilePicture';
-import ProfileName from '../../components/CharacterProfile/ProfileName';
-import ProfileHeader from '../../components/CharacterProfile/ProfileHeader';
-import CommandListBanner from '../../components/CharacterProfile/CommandListBanner';
-import MoveList from './MoveList';
 import FilterMenuContainer from './FilterMenuContainer';
 import Drawer from 'react-native-drawer';
-import CustomText from '../../components/CustomText/CustomText';
-import SearchBar from '../../components/SearchBar/SearchBar';
 import { charProfileNavHeader } from '../../components/NavigationBar';
-import FrameDataRow from '../../components/Spreadsheet/FrameDataRow';
+import AttackList from './AttackList';
```

Comment:
We have a MoveList component and now an AttackList component. This gets confusing since all this component is, is a wrapper.  So naming it as such would make more sense. 

However, I don't necessarily think that moving everything out of the charProfile index this way does much good, since you're not going for the whole force re-render the entire view on orientation approach anymore , but sectioning things out as charProfileHeader, CharProfileSubHeader (containing searchbar), either into separate render methods within index, or separate components would make more sense